ddp-workspace/projects/ddp-lms/src/app/app.module.ts

```diff
@@ -203,7 +203,7 @@ export class AppModule {
     private router: Router) {
     this.renderer = rendererFactory.createRenderer(null, null);
     this.router.events.subscribe(event => {
-      if (event instanceof NavigationEnd) {
+      if (localStorage.getItem('session_key') == null && event instanceof NavigationEnd) {
         if (event.url !== 'undefined' &&
```

Comment:
zim:
Maybe we can avoid looking for specific static page links but its ok to be extra safer and left it in there